WordPressAuthenticator/Unified Auth/View Related/Site Address/SiteCredentialsViewController.swift

```diff
@@ -33,6 +35,20 @@ class SiteCredentialsViewController: LoginViewController {
 		configureSubmitButton(animating: false)
     }
 
+	override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+
+        configureSubmitButton(animating: false)
```

Comment:
This call already exists in `viewDidLoad()`, so maybe it should be removed from here?

Reply:
Removed them both for now. This PR was meant to focus on the username and password textfields. I shouldn't have put in the configuration method for the Continue button.